.travis.yml

```diff
@@ -313,6 +313,10 @@ script:
   - |-
     echo "using $TOOLSET : : $COMPILER : <cxxflags>-std=$CXXSTD ;" > ~/user-config.jam
   - ./b2 libs/ratio/test toolset=$TOOLSET
+  - mkdir __wsl__ && cd __wsl__
```

Comment:
Just for curiosity, what is this __wsl__? What it stands for?

Reply:
Stupid me - that should be `__build__ `
(the name doesn't matter,  but I was testing this with the windows subsystem for linux) will fix.